### PR TITLE
Add new "testing" group

### PR DESCRIPTION
**Pull request by Zac-HD**
Featuring Pytest, Hypothesis, and Hypothesis' Ghostwriter CLI.

### CODE REVIEW

File: groups.yaml [hunk 1]
```diff
@@ -70,3 +70,10 @@
     - GeoPandas
   shape: tri
   cmap: BrBG
+- name: Testing
+  items:
+    - Pytest
+    - Hypothesis
+    - Ghostwriter
+  shape: tri
+  cmap: rainbow  # *not* jet
```
Comment (Carreau) [inline groups.yaml:79]: Lol

File: groups.yaml [hunk 2]
```diff
@@ -70,3 +70,10 @@
     - GeoPandas
   shape: tri
   cmap: BrBG
+- name: Testing
+  items:
+    - Pytest
+    - Hypothesis
+    - Ghostwriter
```
Comment (Carreau) [inline groups.yaml:77]: I'm guessing you are maintainer of ghostwriter ?

Comment (Zac-HD) [reply groups.yaml:77]: Yeah, it's a CLI tool shipped with Hypothesis.

**Comment by Carreau** [comment]
Extra suggestions from slack
```
Pytest (testing 1/3)
>>> def test_spam(): ... / >>> def test_eggs(): ... / $ pytest my/tests/
"Just test something. Going from zero tests to one test is an infinite improvement."
"My needs are usually simple: running a collection of test functions. I don’t take advantage of the plugins, or fixtures, or any of the other advanced features."
Hypothesis (testing 2/3)
>>> from hypothesis import given
Find more bugs with less work, by generating diverse inputs for your tests.
"It would have been impossible to implement a trustworthy autograd library without Hypothesis."
"We used to hand-craft Dataframes...  It's trivial to generate them with Hypothesis, and our coverage is now close to 90%."
"The only problem is that it's too good at finding bugs, so I can't sneak broken code in before feature freeze."
Ghostwriter (testing 3/3)
$ hypothesis write numpy.matmul / $ hypothesis write gzip.compress
Don't like writing tests, or not sure how to start using Hypothesis?  Who you gonna call?  Ghostwriter!
"Is that AI?" "Well, do if-statements and regular expressions count?"
"Sometimes, magic is just someone spending more time on something than anyone else might reasonably expect. --Teller"
```

**Pull request merged**